### DISCUSSION TITLE
Which versions are supported by aioquic?

### DISCUSSION THREAD

**Discussion by lastpepole** [aiortc/aioquic#463] [Q&A]
how to set version number in examples/http3_client.py and examples/http3_server.py.
for example, we want to test h3-29 and h3-v1 version.

**Comment by jlaine** [comment]
`aioquic` supports QUIC draft-29 up to v1, though I imagine we shall at some point drop anything prior to v1 to make room for v2 drafts.

I'm not aware of an `h3-v1` ALPN protocol, as far as I know the ALPN for the final HTTP/3 version is just `h3`.
